pl_bolts/models/detection/faster_rcnn/__init__.py

```diff
@@ -0,0 +1,4 @@
+from pl_bolts.models.detection.faster_rcnn.backbones import create_fasterrcnn_backbone
+from pl_bolts.models.detection.faster_rcnn.faster_rcnn_module import FasterRCNN
```

Comment:
Also here, I don't know why `flake8` is happy about these lines without `# noqa: F401` cc: @Borda

Reply:
Same commoent, __all__ blocks flake8 :sweat_smile: